### PR TITLE
Feature/seed database cr

### LINKED ISSUE #5: Transfer compiled data to postgresSQL db using Knex

### LINKED ISSUE #16: Seed data files using Knex

### PR DESCRIPTION
**Pull request by Fordo29**
## What Features are Being Added?
Add seed data into the database for books

## What is Being Refactored?
Our data that we seeded needed column names refactored to be snake case. 

## Where should the reviewer start?
In seeds directory - books_favorites.js file

## How should this be manually tested?
check inside pgAdmin that all the data populated in the database

## What are the relevant tickets?
Seed data ticket

## Any Other Notes/Questions/Problems?
Not at this time

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ module.exports = {
     },
     migrations: {
       directory: __dirname + '/migrations'
+    },
```
Comment (cagallo) [inline knexfile.js:21]: Needed to manually add this as well as migrations: doesn't get set up with knex. Good to know for future reference.

File: seeds/books_favorites.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> } 
+ */
+const booksData = require('../data/filtered-books-data');
+const favorites = require("../data/favorites-data");
+
+exports.seed = function(knex) {
```
Comment (krdurrett) [inline seeds/books_favorites.js:8]: Great, simple logic to seed the database!

File: seeds/books_favorites.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  // Deletes ALL existing entries
+    return knex('books').del()
+    .then(() => {
+      return knex('books').insert(booksData)
```
Comment (ectweitmann) [inline seeds/books_favorites.js:12]: Nice work Christine! Let's keep the ball rolling 👍🏻

**Review by ectweitmann — APPROVED** [review]

**Review by krdurrett — APPROVED** [review]
This looks great Christine!

File: seeds/books_favorites.js (hunk 2 above)
Comment (cagallo) [inline seeds/books_favorites.js:8]: Clean and it works! We're seeding data!

**Pull request merged**